crates/tests/src/integration/masp.rs

```diff
@@ -31,6 +32,653 @@ use crate::e2e::setup::constants::{
 };
 use crate::strings::TX_APPLIED_SUCCESS;
 
+/// Test interacting with masp amounts that span more than 64 bits.
```

Comment:
do you mean you want the test to enable rewards? as long as the generated rewards themselves never overflow 64 bits for any given note, it should be ok to test this. otherwise, we saw that shit can start going wrong real fast when your rewards overflow

Reply:
In 67f720bfe5495223c98e4a032f7cebbbd2813704 I added some reward checks to `values_spanning_multiple_masp_digits`.

This caught an issue... It seems we are not receiving any NAM rewards for these assets. If I change the shielded tokens to an amount that only spans the first 64 bits, I do get rewards... Wtf?

Reply:
Changing the shielded amount to `1 << 64`, which spans a single masp digit, but occupies `pos=1` (rather than `pos=0`), the test also fails.

Reply:
nevermind. the inflation that was being computed was just too small